keps/0006-stable-kafka-example.md

```diff
@@ -104,6 +105,10 @@ As a Kafka administrator I want to change the number of Kafka brokers in a clust
 
 As a Kafka administrator I want to deploy configuration changes to the cluster with minimal disruption to clients so I can make changes without creating a new cluster.
 
+#### Handle Failures
+
+When a Pod corresponding to Kafka Broker is restarted (because of a Kubelet failure, for example), I want a task to run to verify the health of the Broker and to perform a maintenance if needed
```

Comment:
dot missing at the end.

Reply:
I think in many ways of a `Plan` as the answer to how failures should be handled. I can also think of more custom logic up to the App Developer, even similar declarative in away on how to define what failure is, what it represents and how it is been dealt with. This could be an entire new component or extension to what is currently provided.

Reply:
Agreed. I'm ok with how this is written in the sense of what @guenter said - a user mechanism of watching for application level events and triggering a plan does not exist today.